wrappers/python/nntile/utils/constructors.py

```diff
@@ -0,0 +1,63 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/nntile/pipeline.py
+# TRaining pipeline of NNTile Python package
+#
+# @version 1.0.0
+
+import numpy as np
+
+from nntile.functions import fill_async
+from nntile.nntile_core.tensor import (
+    Tensor_bool,
+    Tensor_fp16,
+    Tensor_fp32,
+    Tensor_fp64,
+    Tensor_int64,
+    TensorTraits,
+)
+
+nnt2np_type_mapping = {Tensor_fp32: np.float32, Tensor_fp64: np.float64}
+
+np2nnt_type_mapping = {
+    np.dtypes.Float32DType: Tensor_fp32,
+    np.dtypes.Float64DType: Tensor_fp64,
+}
+
+
+def astensor(A: np.array, basetile_shape: tuple = None, mpi_distr=[0], next_tag=0):
+    A_traits = TensorTraits(A.shape, basetile_shape if basetile_shape else A.shape)
+
+    A_value = np2nnt_type_mapping[type(A.dtype)](A_traits, mpi_distr, next_tag)
+
+    A_value.from_array(A)
+    return A_value
+
+
+def asarray(tensor_nnt):
```

Comment:
Not sure about naming. It is really confusing to see in the code `asarray` and `astensor` in adjacent lines: what is the difference between them? This is why I am commenting this module and found that it missing in #62.

From my perspective, you should choose a name for `Tensor_*` creation routine between `asarray` (common choice) and `astensor` (PyTorch-only choice). Then rename the current `asarray`. The name should be clear from a context that this function converts a `Tensor_*` to `np.ndarray`. This routine will be needed until  `Tensor_*` support Buffer Protocol.

Use all routines defined here with some module specifier like `nntile.zeros` or `nnt.zeros` or whatever, please. Sometimes it is hard to understand what `zeros` is actually used.

Reply:
asnntile to convert into nntile
asnumpy to convert to numpy